src/Card/CardHeader.js

```diff
@@ -47,9 +48,9 @@ const CardHeader = props => {
     }
 
     return (
-        <div >
+        <Wrapper >
```

Comment:
You can use `<React.Fragment>` build-in HOC of the lib if you need to just wrap up several elements. Or, now blank tag also becomes very popular `<></>`